dist/pom.xml

```diff
@@ -54,7 +54,8 @@
         </noSnapshot.buildvers>
         <snapshot.buildvers>
             314,
-            332
+            332,
+            340
```

Comment:
since 340 shim cannot pass the build for now, please do not add it to the snapshot list property,
as our nightly build will include all noSnapshot + snapshot build, if failed the snapshot jar would not be deployed to remote.

I would prefer to hardcode this in test itself before we could enable regular build for it

Reply:
Thanks @pxLi for letting me know the issue if we add 340 here. I have removed it from pom and added in yml file. PTAL